### PR TITLE
@smit4297

### PR DESCRIPTION
**Pull request by smit4297**
I have created this beautiful login page.
 - [X] I have read the Code Of Conduct.
 - [X] I have followed all the steps of submission properly.

### CODE REVIEW

**Review by harshptl14 — APPROVED** [review]
Good work!

**Comment by yashpaneliya** [comment]
Hey @smit4297  , Thanks for submitting PR.

Please change the comment according to [CONTRIBUTING.md](https://github.com/clubgamma/Awesome-Flutter-Art/blob/master/CONTRIBUTING.md). Then good to go.

Tweet about your contribution  and tag @clubgamma with #hacktoberfest2020.

Then wait for our team to review and merge your PR.

**Comment by yashpaneliya** [comment]
Hello contributor👋,

As we all know that Hacktoberfest-2020 is ended. Hope you enjoyed it a lot and made some awesome contributions. 
So now it's time for schwags🎁🤩

**Send your email (linked with your GitHub account) in the reply of this comment. Mention any one of the maintainers (@yashpaneliya @rudrabarad @harshptl14 @chirayu-joshi @dhruvpatel1706 ) in the reply.**

**We'll send a form on the email you'll provide. Please reply ASAP.**

Join [Club Gamma Discord](https://discord.com/invite/kjnp6wU) for future updates.

**Pull request merged**
